app/components/address-mapper.tsx

```diff
@@ -1,20 +1,8 @@
 "use client";
 
 import { useEffect, useState } from "react";
-import {
-  Box,
-  Flex,
-  Heading,
-  Text,
-  List,
-  ListItem,
-  Image,
-  HStack,
-  Link,
-  useToast,
-} from "@chakra-ui/react";
-import NextLink from "next/link";
-import { HeadingProps } from "./heading";
+import { Box } from "@chakra-ui/react";
+import { toaster } from "@/components/ui/toaster";
```

Comment:
Chakra's Toast component is now an auto-gen one that gets edited and committed to users' repos.